exercises/conor_finn/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,167 @@
+#include "main.h"
+#include <algorithm>
+#include <functional>
+
+using std::cin;
+using std::cout;
+using std::endl;
+using std::ifstream;
+using std::map;
+using std::ofstream;
+using std::pair;
+using std::string;
+using std::vector;
+
+/**
+ * @brief Creates a results file containing the number of
+ * counts of words of >4 characters in a user defined file.
+ *
+ * @return int 0 on success, >0 otherwise.
+ */
+int wordCounterInterface() {
+  // Number of characters a word must have to be counted.
+  const int characters = 5;
+  // Ask for a file path.
+  cout << "Please enter the file path:";
+  string path;
+  cin >> path;
+  ifstream filein;
+  filein.open(path.c_str());
```

Comment:
An alternative to the above two lines is to open the file in the initializer, i.e.: `ifstream fileIn {path.c_str()};`. Also `filein`'s second "I" should be capitalized